libs/perun/services/src/lib/sponsored-members-pdf.service.ts

```diff
@@ -1,53 +1,84 @@
 import { Injectable } from '@angular/core';
 import { PDFService } from './pdf.service';
 
+interface MemberData {
+  name: string;
+  status: string;
+  login?: string;
+  password?: string;
+}
+
+interface MemberDataCell {
+  text?: string;
+  colSpan?: number;
+  rowSpan?: number;
+  alignment?: string;
+  border?: boolean | boolean[];
+  margin?: number[];
+  fontSize?: number;
+  bold?: boolean;
+  color?: string;
+}
+
+interface MemberDataTable {
+  heights: number[];
+  widths: number[];
+  body: MemberDataCell[][];
+}
+
+export interface MemberDataTableWrapper {
+  table: MemberDataTable;
+}
```

Comment:
Do you need this interface if it is the same as `MemberDataTable` just only hidden under another attribute?